### ISSUE TITLE
[csrng] invalid/unused commands

### ISSUE THREAD

**Issue by cindychip** [lowRISC/opentitan#15947]
CSRNG declared `acmd_e` with 8 values, but I believe only 5 of them are valid acmd.
https://github.com/lowRISC/opentitan/blob/master/hw/ip/csrng/rtl/csrng_pkg.sv#L37

As far as I understand:
1). In CSRNG testbench: it won't drive this invalid commands:
https://github.com/lowRISC/opentitan/blob/master/hw/ip/csrng/dv/cov/csrng_cov_if.sv#L57
But I wonder if we should try these error inputs and see if CSRNG response correctly?

2). In EDN testbench: I do not think we have any check for it. Maybe we could add some assertions to check the `acmd` won't return these invalid values?

FYI: @johngt

**Comment by johngt** [comment]
This one slipped the radar as it was tagged as M3.
@vogelpi / @andreaskurth - could you please respond to @cindychip 's questions.
If there is something that we need to do related to CSRNG for M2 coming out of this please tag me again and we will change to M2 etc.

**Comment by vogelpi** [comment]
Thanks @johngt, @marnovandermaas had asked a very similar question in #16203.

For CSRNG, it seems that only the 3 LSBs are being looked at at the moment. I agree we should test this and it seems the agent has now support for it thanks to @cindychip . However, I see it much less critical than other things we still need to verify/hit in CSRNG DV for M2. I don't think we get much out of this in terms of coverage/critical functional bugs at the moment.

M3 seems right for this one.

**Comment by vogelpi** [comment]
Actually, we ended up testing this as part of V2/V2S. IIRC @marnovandermaas implemented this. The `csrng_alert` test now also writes invalid commands and ensures the main FSM signals a recoverable alert and halts. The test then disables/re-enables CSRNG to push in another command. So this is actually done and I am thus closing the issue.

We have now filed a follow-up issue to define what the correct behavior of CSRNG should be and whether an invalid command should also be signaled in the status bit, e.g. back to EDN. See #16516. For EDN itself, I don't think we need to do anything. Because the different command values have no meaning for EDN itself. EDN just forwards the commands to CSRNG and thus I believe shouldn't check them.
